lib/tests/common.go

```diff
@@ -0,0 +1,144 @@
+package test
+
+import (
+	"archive/tar"
+	"bytes"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/appc/docker2aci/lib/internal/typesV2"
+)
+
+type Layer map[*tar.Header][]byte
```

Comment:
This will use tar.Header pointer values as keys, not the tar.Header values. If put two equal values of `tar.Header`, but with distinct memory locations, they will have a different key.